3_javascript/javascript1/week2/week2.js

```diff
@@ -0,0 +1,148 @@
+// Flight booking fullname function
+
+function getFullname(firstname, surname, useFormalName) {
+    if (useFormalName) {
+        return ('Lord ' + firstname + " " + surname);
+
+    } else {
+        return (firstname + " " + surname);
+    } 
+}  
+
+const fullname1 = getFullname("Benjamin", "Hughes", true);
+const fullname2= getFullname("Benjamin", "Hughes", false);
+
+console.log(fullname1, fullname2);
+
+
+// Event application
+
+function getEventWeekday(eventy) {
+    const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+    const dateNow = new Date();
+    
+    const numberDayNow = dateNow.getDay();
+ 
+    const nextDate = (numberDayNow + eventy) % 7;
+    console.log('event on', weekday[nextDate]);
+
+}
+
+getEventWeekday(9);
+
+// Weather wear
+
+function whatToWear(temp) {
+    let weather;
+    let clothes;
+    if (temp<0) {
+         weather = 'cold';
+    } else if (temp>=0 && temp<=7) {
+         weather = 'not very cold';
+    } else if (temp>7 && temp<=15) {
+         weather = 'warm';
+    } else if (temp > 15 && temp<=25) {
+         weather = 'hot';
+    } else {
+         weather = 'very hot';
+    };
+
+    switch (weather) {
+        case 'cold': clothes= 'down jacket and hat';
+        break;
+        case 'not very cold': clothes= 'only jacket';
+        break;
+        case 'warm': clothes= 'jeans suit';
+        break;
+        case 'hot': clothes= 'shorts and a t-shirt';
+        break;
+        case 'very hot': clothes= 'do not forget swimming suit';
+        break;
+        default: clothes = 'do not forget wear shoes'
+    }
```

Comment:
Nice! I like the structure, but I think that one of the condition is redundant, you can already assign the clothes value in the first condition I can see :) 

```
if (temp < 0) {
    weather = 'cold'
    clothes= 'down jacket and hat';
  } else if (temp >= 0 && temp <= 7) {
    weather = 'not very cold'
    clothes= 'only jacket';
  } else if (temp > 7 && temp <= 15) {
    weather = 'warm'
    clothes = 'jeans suit'
  } else if (temp > 15 && temp <= 25) {
    weather = 'hot'
    clothes= 'shorts and a t-shirt';
  } else {
    weather = 'very hot'
    clothes= 'do not forget swimming suit';
  }
```

Reply:
yes, i've want tried to use 'switch'

Reply:
okok :) But also important is to make code as clean as possible, therefore, it is better to use just one check if its possible  (e.g my example above), anyway good job!:)